docs/tasks/configure-pod-container/security-context.md

```diff
@@ -0,0 +1,351 @@
+---
+assignees:
+- erictune
+- mikedanese
+- thockin
+title: Configure a Security Context for a Pod or Container
+redirect_from:
+- "/docs/user-guide/security-context/"
+- "/docs/concepts/policy/container-capabilities/" 
+---
+
+{% capture overview %}
+
+A security context defines privilege and access control settings for
+a Pod or Container. Security context settings include:
+
+* Discretionary Access Control: Permission to access an object, like a file, is based on
+[user ID (UID) and group ID (GID)](https://wiki.archlinux.org/index.php/users_and_groups).
+
+* [Security Enhanced Linux (SELinux)](https://en.wikipedia.org/wiki/Security-Enhanced_Linux): Objects are assigned security labels.
```

Comment:
I feel like it's worth mentioning AppArmor and Seccomp here as well. Technically those are annotations, but they will be moved to fields on the security context soon.